master/getting-started/kubernetes/installation/index.md

```diff
@@ -115,14 +115,21 @@ cat >/etc/cni/net.d/10-calico.conf <<EOF
     },
     "policy": {
         "type": "k8s"
+    },
+    "kubernetes": {
+        "kubeconfig": "</PATH/TO/KUBECONFIG>"
     }
 }
 EOF
 ```
 
 Replace `<ETCD_IP>:<ETCD_PORT>` with your etcd configuration.
+Replace `</PATH/TO/KUBECONFIG>` with your kubeconfig file. See [kubernetes kubeconfig](http://kubernetes.io/docs/user-guide/kubeconfig-file/) for more information
+about kubeconfig.
 
-For more information on configuring the Calico CNI plugins, see the [configuration guide](https://github.com/projectcalico/calico-cni/blob/v1.4.1/configuration.md)
+If your kube-apiserver is configured with authentication like client certificates, bearer tokens, or HTTP basic auth, you also
```

Comment:
This is looking better.  A couple of minor tweaks:

> is configured with authentication like client certificates, bearer tokens, or HTTP basic auth

This can all be configured through the kubeconfig file, so rather than say "needs to specified" I'd change the wording to be something like "can also be specified directly in the CNI config file".

> See the [configuration guide]

"See" shouldn't be capitalized, and this line should end in a `.`